### ISSUE TITLE
Commit count seems to be off

### ISSUE THREAD

**Issue by LTMenezes** [RhysSullivan/shiptalkers#43]
Hey, awesome project by the way!

The commit count seems to be way off for me, it counted 263 commits for me but I had more than 800 on 2018 alone.
Link to my shiptalkers' page: https://shiptalkers.dev/compare?name=ltmenezes

Was it an issue because of the current load or is there any filter that I'm not aware of? E.g. only counts public repos owned by you.

**Comment by timneutkens** [comment]
(Disclaimer: I'm not the maintainer of this site)
The tool only counts the last year, which is what is pulled from your GitHub profile it seems.
- Your private contributions are not included if they're not on your GitHub profile.
- You can try opening https://github.com/ltmenezes in an incognito tab 

So contributions you did in 2018 are not counted, similarly it only looks at the last year of tweets, not all years of tweets.

**Comment by raulillana** [comment]
My commit count is also very low because of this. But counting the commits in private and organization repos would need authentication, and that will be a security / reliability issue probably…

Maybe GitHub should make those data available in some way through the API. 😅

**Comment by LTMenezes** [comment]
Thanks @timneutkens, that makes sense!

I will close the issue for now then, cheers.

**Comment by raulillana** [comment]
@timneutkens yeah, had it active for like a decade, thanks. Does it scrape from the graph or from the API then?
